java/arcs/core/data/CreateableStorageKey.kt

```diff
@@ -36,12 +37,10 @@ data class CreateableStorageKey(
         const val CAPABILITY_ARG_SEPARATOR = "?"
 
         private val CREATEABLE_STORAGE_KEY_PATTERN =
```

Comment:
Do you have tests for this regex? I would even accept a comment with a link to a regexr page.

Reply:
There are tests for parsing using this regex, which is largely what this PR does.

I am not confident that we should put a link to regexr page next to every regex in our codebase. Regex is widely considered something that engineers need to be somewhat familiar with.